grafana-plugin/src/module.ts

```diff
@@ -1,15 +1,35 @@
 import { ComponentClass } from 'react';
 
-import { AppPlugin } from '@grafana/data';
+import { AppPlugin, PluginExtensionPoints } from '@grafana/data';
 
+import MobileAppConnection from 'containers/MobileAppConnection/MobileAppConnection';
 import PluginConfigPage from 'containers/PluginConfigPage/PluginConfigPage';
 import { GrafanaPluginRootPage } from 'plugin/GrafanaPluginRootPage';
 
 import { OnCallPluginConfigPageProps, OnCallPluginMetaJSONData } from './types';
 
-export const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
+const plugin = new AppPlugin<OnCallPluginMetaJSONData>().setRootPage(GrafanaPluginRootPage).addConfigPage({
   title: 'Configuration',
   icon: 'cog',
   body: PluginConfigPage as unknown as ComponentClass<OnCallPluginConfigPageProps, unknown>,
   id: 'configuration',
 });
+
+if (
+  'configureExtensionLink' in plugin &&
+  PluginExtensionPoints != null &&
+  'ExploreToolbarAction' in PluginExtensionPoints
+) {
+  plugin.configureExtensionComponent({
```

Comment:
here is [the only example I could find](https://github.com/grafana/grafana-pdc-app/blob/e39bd2f2ca6cf5cf1c11cd96805f2dfd6b08b505/src/module.ts#L37) of another plugin using `configureExtensionComponent`